src/Components/Web3/Contracts/DBETToVETDepositContract.ts

```diff
@@ -1,44 +1,38 @@
 /* eslint-disable no-console */
 import BaseContract from './BaseContract'
 import { BigNumber } from 'bignumber.js'
-import { Subject, interval, Observable } from 'rxjs'
-import {
-    mergeMap,
-    timeout,
-    filter,
-    catchError,
-    tap,    
-} from 'rxjs/operators'
+import { Subject, Observable } from 'rxjs'
+import { timeout, filter, catchError, tap } from 'rxjs/operators'
 import { Config } from '../../Config'
 import Helper from '../../Helper'
 import Web3 from 'web3'
-import { Contract } from 'web3/types'
 import { VETDeposit } from './Deposit'
+import Contract from 'web3/eth/contract'
 
 const ethAbi = require('web3-eth-abi')
 const Contract_DBETToVETDeposit = require('../../Base/Contracts/DBETToVETDeposit.json')
```

Comment:
Are we still using this import? Shouldn't it be replaced with the "contracts" import below?